examples/misc/raptor.py

```diff
@@ -1,5 +1,34 @@
 #!/usr/bin/env python3
 
+'''
+Demonstrate the "raptor" features for remote Task management.
+
+This script and its supporting files may use relative file paths. Run from the
+directory in which you found it.
+
+Refer to the ``raptor.cfg`` file in the same directory for configurable run time
+details.
+
+By default, this example uses the ``local.localhost`` resource with the
```

Comment:
In such a context, how should the reader/user interpret "nodes", with respect to `cores_per_node`, `nodes_per_worker`, etc., and the comment "Each master uses a node, and each worker also uses a node."?